OutOfSchool/OutOfSchool.WebApi/Services/ProviderService.cs

```diff
@@ -467,7 +467,7 @@ await SendNotification(checkProvider, NotificationAction.Update, statusChanged,
         licenseChanged = false;
 
         if (!(checkProvider.FullTitle == providerDto.FullTitle
-              && checkProvider.EdrpouIpn == long.Parse(providerDto.EdrpouIpn)))
+              && checkProvider.EdrpouIpn == providerDto.EdrpouIpn))
```

Comment:
Maybe it's better to use `checkProvider.EdrpouIpn.Equals(providerDto.EdrpouIpn)`

Reply:
Okay, after some tests I've found that `==` can be enough for this case, so this comment no longer useful.